examples/applications/plot_cyclical_feature_engineering.py

```diff
@@ -209,9 +209,10 @@
             ("categorical", ordinal_encoder, categorical_columns),
         ],
         remainder="passthrough",
-    ),
+        verbose_feature_names_out=False,
+    ).set_output("pandas"),
```

Comment:
Although this is valid, do you think we should prefer calling `set_output` on the pipeline?

In other words:

```python
gbrt_pipeline = make_pipeline(
    ...
).set_output(transform="pandas")
```

Reply:
Ok, let's be fancy, I will update the PR.

Reply:
Done.